src/v2/Apps/Auctions/Components/AuctionsMeta.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react"
+import { Meta, Title } from "react-head"
+import { getENV } from "v2/Utils/getENV"
+
+export const AuctionsMeta: React.FC = () => {
+  return (
+    <>
+      <Title>AuctionsApp App | Artsy</Title>
+      <Meta property="og:title" content="Example App" />
+      <Meta name="description" content="Fill this with a proper description" />
```

Comment:
Unless you'll do a very very fast followup PR I think it's worth getting an acceptable, user-facing description since these will show up in search results and in social media links.

Reply:
Ah, we're still pointed at `/auctions2` -- there's no way to get to this page. Route update will come in followup (updated description)